Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1333,6 +1333,8 @@ extension BrowserViewController: MenuActionDelegate {
             switch menuAction {
             case .openNewNormalTab:
                 self.openURLInNewTab(nil, isPrivate: false, isPrivileged: true)
+                LeanplumIntegration.sharedInstance.track(eventName: .openedNewTab)
```

Comment:
Note that this is not the only place where people can open new tabs. We also have the + button in the toolbar, which goes through a different path I think? I am not sure. May be worth to check.